app/controllers/friendships_controller.rb

```diff
@@ -0,0 +1,11 @@
+class FriendshipsController < ApplicationController
+  def destroy
+    if user = User.find_by(id: params[:user_id])
```

Comment:
I think there's something odd here. We're using `if` while we're assigning user to `User.find_by(id: params[:user_id])`